clients/typescript/src/types.ts

```diff
@@ -69,6 +139,43 @@ interface PathfinderUniqueParams {
   num_threads: number;
 }
 
+/**
+ * @typedef {Object} PathfinderParams
+ * Parameters for the Pathfinder algorithm.
+ * @property {string | StanVariableInputs} [data=""] The data for the model.
+ * @property {number} [num_paths=4] The number of individual paths to run.
+ * @property {string | StanVariableInputs | string[] | StanVariableInputs[]} [inits=""]
+ * The initial values for the algorithm. If an array, must have length `num_paths`.
+ * @property {number | null} [seed] The seed for the random number generator.
+ * If unspecified, a random seed will be generated.
+ * @property {number} [id=1] The ID for the first path.
```

Comment:
(And the others will all be incremented from there?)

Reply:
Yes. In practice, almost no users will need to set this